tests/test_nc_to_mmd.py

```diff
@@ -12,50 +12,214 @@
 import pathlib
 import tempfile
 import yaml
-import datetime
 import warnings
 import unittest
 
-import pandas as pd
-
-from unittest.mock import patch
+from dateutil.parser import isoparse
 from filehash import FileHash
 from lxml import etree
 from netCDF4 import Dataset
 from pkg_resources import resource_string
+from unittest.mock import patch
 
-from py_mmd_tools.nc_to_mmd import Nc_to_mmd
-from py_mmd_tools.yaml_to_adoc import nc_attrs_from_yaml, get_attr_info
+from py_mmd_tools.nc_to_mmd import Nc_to_mmd, validate_iso8601
+from py_mmd_tools.yaml_to_adoc import nc_attrs_from_yaml
```

Comment:
Can this be shortened?

Reply:
not sure what you want. Maybe you could use the suggestion mode? Anyway, I'll just merge since this is not critical...